### ISSUE TITLE
Add Routes : Where is electrode-houseparty-example-component ?

### ISSUE THREAD

**Issue by belemaire** [electrode-io/electrode-io.github.io#135]
In [Add Routes](https://electrode-io.github.io/docs/server_config.html) page code for the component, it references `electrode-houseparty-example-component` (`import { HouseParty } from "electrode-houseparty-example-component";`).

This component does not exists so when I reach the [Server Config](https://electrode-io.github.io/docs/server_config.html) page, and I run `gulp hot`, the command fails :

```
ERROR in ./client/routes.jsx
Module not found: Error: Cannot resolve module 'electrode-houseparty-example-component' in /Users/blemair/Code/my-electrode-app/client
 @ ./client/routes.jsx 14:43-92
```

**Comment by belemaire** [comment]
Found out the issue.

Related to https://github.com/electrode-io/electrode-io.github.io/issues/129

Seems like the `npm install` command line reference in https://github.com/electrode-io/electrode-io.github.io/issues/129 should be for this component. Please see https://github.com/electrode-io/electrode-io.github.io/issues/129 for reference

**Comment by tiffine-koch** [comment]
resolved in PR #144 - updates add_routes.md to give clear instructions on using the separate/user generated published module inside of app
